src/Components/ArtistFilter/Filter.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Subscribe } from "unstated"
+
+import { FilterState } from "./state"
+import ArtworksContent from "./Artworks"
+
+class Filter extends React.Component<RelayProps, null> {
+  renderCurrentlySelected(filter, state) {
+    if (
+      (filter === "institution" || filter === "gallery") &&
+      state.partner_id
+    ) {
+      return state.partner_id
+    }
+    if (filter === "major_period" && state.major_periods) {
+      return state.major_periods[0]
+    }
+    return state[filter]
+  }
+
+  renderFilters(filters) {
```

Comment:
This renders all our filters, and displays the currently selected filters (if there are any).